www/src/examples/Dropdown/AutoClose.js

```diff
@@ -0,0 +1,15 @@
+<>
+  {[true, 'outside', 'inside', false].map((autoClose, index) => (
```

Comment:
Wonder if we should rearrange/reword the examples so it mirrors upstream Bootstrap examples:
https://getbootstrap.com/docs/5.0/components/dropdowns/#auto-close-behavior

I got thrown off with the autoclose behavior for "inside" and "outside" differing from Bootstrap's docs, but turns out it was an issue of ordering/wording.